lib/mount_util.c

```diff
@@ -81,13 +91,45 @@ static int mtab_needs_update(const char *mnt)
 }
 #endif /* IGNORE_MTAB */
 
+static int fuse_mount_posix_spawn_attr(posix_spawnattr_t *attr, const char *cmd,
+				       sigset_t mask)
+{
+	int res;
+
+	posix_spawnattr_init(attr);
+
+	int flags = POSIX_SPAWN_RESETIDS | POSIX_SPAWN_SETSIGMASK;
+	res = posix_spawnattr_setflags(attr, flags);
+	if (res) {
```

Comment:
Why the difference between this check and the one on line 110? From the man pages both appear to return 0 on success and an errno on error.

Reply:
Thanks,  the check below for (res < 0) needs to be fixed. Although I see it in line 103